cluster_health_check.md

```diff
@@ -0,0 +1,88 @@
+Etcd Cluster Health Check
+========
+
+Synopsis
+========
+
+```nginx
+http {
+    # required declares a shared memory zone to store endpoints's health status
+    lua_shared_dict healthcheck_shm 1m;
+
+    server {
+        location = /healthcheck {
+            content_by_lua_block {
+                local etcd, err = require "resty.etcd" .new({
+                    protocol = "v3",
+                    http_host = {
+                        "http://127.0.0.1:12379", 
+                        "http://127.0.0.1:22379",
+                        "http://127.0.0.1:32379",
+                    },
+                    user = 'root',
+                    password = 'abc123',
+
+                    # the health check feature is optional, and can be enabled with the following configuration.
+                    health_check = {
+                        shm_name = 'healthcheck_shm',
+                        fail_timeout = 1,
+                        max_fails = 1,
+                        disable_duration = 100
```

Comment:
Actually in Nginx the disable duration is same as fail_timeout, maybe we can follow this design.

@membphis What do you think?

Reply:
agree with you @tokers

Reply:
got